### DISCUSSION TITLE
DGT Centaur sensors.

### DISCUSSION THREAD

**Discussion by Jasiek-dev** [EdNekebno/DGTCentaurMods#26] [General]
Does anyone know what kind of sensors are installed in DGT Centaur?

**Comment by wormstein** [comment]
As I was saying yesterday, pieces of Centaur have an aluminium foil underneath. Based on this I would say it's one type of these innductive proximity sendors like here: https://www.keyence.co.uk/ss/products/sensor/sensorbasics/proximity/info/

I let someone else to give a second opinion.

**Comment by EdNekebno** [comment]
With a quick test and without taking the board to pieces, I believe it's the capacitive type. If I run our code for showing pieces on the board and then stick my thumb on the board (no metal near my thumb) - then it shows up as a piece.

The question was for a different reason, so I'm going off topic, but of course this makes we wonder how sensitive they are and if half the foil would return half the single. i.e. when it comes to piece detection - can it not do it, or did DGT just not put it in the centaur process? I'd guess it can't - but it's an interesting thought.

**Comment by wormstein** [comment]
This explains why I was triggering the board to beep when I was moving it around and touching the fields.
But if it's capacitive why using a board piece on my smartphone's screen don't work.

**Reply by EdNekebno** [reply]
Mine the same. But also ignore me - it could have been pure chance. If I try now then it doesn't do it.
